docs/jwst/datamodels/index.rst

```diff
@@ -1,16 +0,0 @@
-.. _data-models:
```

Comment:
Would it be possible to retain at least some single, simple rst file in this location within jwst that just points/links over to the top-level datamodels docs that will now be over in `stdatamodels`?  I don't like the idea of folks having to manually go looking for them in some other package, because they are an integral part of the jwst pipeline.

Reply:
Yes, but this will require a release of `stdatamodels` with the datamodels present so that I can build intersphinx links.